app/index/post.controller.js

```diff
@@ -0,0 +1,8 @@
+'use strict'
+
+// Local dependencies
+const confirmation = require('./../confirmation')
+
+module.exports = (req, res) => {
```

Comment:
Same here, post controller? not sure what it is relating too, but I guess is just to pass the pages through for now isn't?

Reply:
See the discussion of https://github.com/alphagov/pay-direct-debit-frontend/pull/17#discussion_r157474689